patient-appointment-management/assets/index.html

```diff
@@ -0,0 +1,147 @@
+    </p>
+  </header>
+  <main>
+    <section>
+      <div class="flow-loader"></div>
+      <div id="flow-deploy" style="display: none;" class="deployer">
+          <h4>Your Studio Flow is not yet deployed</h4>
+          <a class="button" onclick="deployStudioFlow(event);">Deploy Studio Flow</a>
+          <div id="flow-loader button-loader" style="display: none;"></div>
+      </div>
+      <div id="flow-deployed" style="display: none;" class="deployer">
+          <h4><span class="checkmark">&#10003;</span> Your Studio Flow is deployed</h4>
+          <a id="flow-open" class="button button-outline" href="" target="_blank">Open Studio Flow</a>
+          <p><b>FLOW REST API URL:</b>&nbsp<i id="flow-rest-api-url"></i></p>
+      </div>
+    </section>
+
+    <section>
+      <div class="aws-bucket-loader"></div>
+      <div id="aws-bucket-deploy" style="display: none;" class="deployer">
+        <h4>Your AWS bucket is not yet deployed</h4>
+        <a class="button" onclick="deployAWSBucket(event);">Deploy AWS Bucket</a>
+        <div id="aws-bucket-loader button-loader" style="display: none;"></div>
+      </div>
+      <div id="aws-bucket-deploying" style="display: none;" class="deployer">
+        <h4>Your AWS bucket in being deployed ...</h4>
+        <p>Please periodically refresh this page until deployment is complete</p>
+      </div>
+      <div id="aws-bucket-deployed" style="display: none;" class="deployer">
+        <h4><span class="checkmark">&#10003;</span> Your AWS bucket is deployed</h4>
+        <a id="aws-bucket-open" class="button button-outline" href="" target="_blank">Open AWS Bucket</a>
+      </div>
+    </section>
+
+    <section>
+      <div class="aws-application-loader"></div>
+      <div id="aws-application-deploy" style="display: none;" class="deployer">
+        <h4>Your AWS application is not yet deployed</h4>
+        <a class="button" onclick="deployAWSApplication(event);">Deploy AWS Application</a>
+        <div id="aws-application-loader button-loader" style="display: none;"></div>
+      </div>
+      <div id="aws-application-deploying" style="display: none;" class="deployer">
+        <h4>Your AWS application in being deployed ...</h4>
+        <p>Please periodically refresh this page until deployment is complete</p>
+      </div>
+      <div id="aws-application-deployed" style="display: none;" class="deployer">
+        <h4><span class="checkmark">&#10003;</span> Your AWS application is deployed</h4>
+        <a id="aws-application-open" class="button button-outline" href="" target="_blank">Open AWS CloudFormation</a>
+      </div>
+    </section>
+
+    <section>
+      <div id="ready-to-use" style="display: none;">
+        <h4><span class="checkmark">&#10003;</span> Patient Appointment Management Application is Ready for Integration</h4>
+
+        <p>
+          In order to complete the EHR integration,
+          you'll need to provide the following information to your EHR integration implementor.
```

Comment:
Consider a link somewhere in this document describing what an EHR is; ordinary developers wanting to use this as an example will appreciate the definition.

Reply:
Hey @jmulcahey-twilio, just curious what you mean by "ordinary developers wanting to use this an example"? @bochoi-twlo you can use this link if you'd like to include: https://www.healthit.gov/faq/what-electronic-health-record-ehr

Reply:
Hey Katie; generally speaking, the apps in this repo end up being used as the basis for new code by developers both within and outside of Twilio. One example of this is the `vaccine-standby` app, bits and pieces of which have seen reuse in many other apps, including this one. Because that's a big use case for the `function-templates` repo, we tend to encourage apps to be as understandable as possible for that target demographic.

Reply:
Ah I see, thanks for the insight. I wasn't thinking about this as a potential component for alternative use-cases in future. I am including EHR reference links in the README also